src/classes/RLLP_OppRollup.cls

```diff
@@ -593,6 +599,10 @@ public without sharing class RLLP_OppRollup {
             
                 //calculate best gift year
                 decimal yrAmt = (decimal)r.get('TotalOppAmount');
+                // Handle currency conversion for the Household in a MultiCurrency org
+                if(RLLP_OppRollup_UTIL.isMultiCurrency()){
```

Comment:
After our coding convention and whitespace talks I noticed the need for whitespace on the 3 conditionals, also I'm wondering if we'd want to move the variable declaration on 607 up so that we can use the variable vs retrieving it from the map multiple times?